receiver/googlecloudspannerreceiver/cardinality.md

```diff
@@ -0,0 +1,75 @@
+# Cloud Spanner: Controlling Metrics Cardinality for Open Telemetry Receiver
+
+## Overview
+This document describes the algorithm used to control the cardinality of the Top N metrics exported by Cloud Spanner receiver in OpenTelemetry collector.
+With cardinality limits enforced in Cloud Monitoring and possibly in Prometheus, it is critical to control the cardinality of the metrics to avoid drops in time series which can limit the usability of Cloud Spanner custom metrics.
+
+## Background
+Cloud Monitoring has 200,000 active time series (streams) [limits](https://cloud.google.com/monitoring/quotas) for custom metrics per project.
+Active time series translates to streams that are sent in the last 24 hours.
+This means that the number of streams that can be sent by OpenTelemetry collector should be controlled to this limit as the metrics will be dropped by Cloud Monitoring.
+
+## Calculation
+For simplicity purpose and for explaining of calculation, which is currently done in implementation, lets assume that we have 1 project, 1 instance and 1 database.
+Such calculations are done when collector starts. If cardinality total limit is different(for calculations below we assume it is 200000) - numbers will be different.
+
+According to the metrics metadata configuration(at the moment of writing this document) we have:
+- 30 low cardinality metrics(amount of total N + active queries summary metrics);
```

Comment:
Metrics isn't my strongest side, care to explain in the readme the difference between Top N vs. Total N?

Reply:
You can find this info in receiver's **README.md**. First paragraph - there is dedicated link to Spanner Sys tables description there.